assets/ui.css

```
@@ -1,3 +1,14 @@
+.close-button {
```

Maybe just to be safe?
```suggestion
.respec-modal .close-button {
```